content/applications/finance/accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -38,49 +33,50 @@ following address:
 
 - https://production.odoofin.com/
 
-First Synchronization
+First synchronization
 ---------------------
 
-You can start synchronization either by going to :menuselection:`Accounting --> Configuration
---> Add a Bank Account` or via the configuration bar on the accounting dashboard.
+You can start synchronization either by going to the Accounting app and
+:menuselection:`Configuration --> Add a Bank Account` or via the bank journal's configuration on the
+dashboard.
 
 Now you can search for your bank institution. Select it and follow the steps to synchronize with it.
 
 .. note::
    If you have any issues during your first synchronization, please verify that your
-   web browser doesn't block pop-ups and that your adblocker is disabled.
+   web browser doesn't block pop-ups and that your ad-blocker is disabled.
 
-During your first synchronization, you will be asked for a phone number to secure your account.
-The reason we ask for such information is that we don't want your data falling into the wrong
-hands. Therefore, if we detect suspicious activity on your account, we block all requests coming
-from your account, and you need to reactivate it using that phone number.
-The third-party provider may request more information in order to connect with your bank institution.
+.. important::
+   When choosing the date for the first bank statement synchronization, pick the date of when you
+   start working on your Odoo accounting database. For example, if you import your closing
+   balance in Odoo on the 31.12.2022 and you start recording accounting transactions on the
+   01.01.2023, your synchronization date should be 01.01.2023.
+
+You must provide a phone number during your first synchronization to secure your account. We ask for
+such information because we don't want your data falling into the wrong hands. Therefore, if we
+detect suspicious activities on your account, we block all requests coming from your account, and
+you need to reactivate it using that phone number.
+
+The third-party provider may request more information in order to connect with your
+bank institution.
 This information is not stored on Odoo's servers.
 
 By default, transactions fetched from an online source are grouped inside the same statement, and
 one bank statement is created per month. You can change the bank statement creation periodicity
 in your journal settings.
 
-You can find all your synchronizations by going to :menuselection:`Accounting --> Configuration -->
+You can find all your synchronizations by going to :menuselection:`Configuration -->
 Online Synchronization`.
```

Comment:
```suggestion
You can find all your synchronizations by going to :menuselection:`Accounting Dashboard --> Configuration -->
Accounting: Online Synchronization`.
```
We usually keep the full path as people might not check the previous sections and jump straight to that info.
Also, I added the sub-menu, but feel free to remove it if you don't like it ;)